src/doc/trpl/structs.md

```diff
@@ -87,3 +87,32 @@ fn main() {
     point.y = 6; // this causes an error
 }
 ```
+
+# Update syntax
+
+A `struct` can include `..` to indicate that you want to update just some
+values of a struct. For example:
```

Comment:
@pnkfelix maybe I'm focused too much on the 'update' part of 'functional update syntax'...